chart/epinio/values.yaml

```diff
@@ -110,8 +114,9 @@ minio:
 
 epinio-ui:
   enabled: true
-  epinioTheme: light
-  epinioVersion: "v1.6.2"
+  epinioUI:
+    theme: light
+    version: "v1.6.1"
```

Comment:
Wrong version here, it should be `v1.6.2`.

@enrichman given that the key names changed don't we need to adapt our CI somehow to be able do the version bump?

Reply:
Good catch. We need to update also this: https://github.com/epinio/helm-charts/blob/main/updatecli/updatecli.d/epinio.yaml#L89

Reply:
@richard-cox please adapt the key name within this PR here https://github.com/epinio/helm-charts/blob/main/updatecli/updatecli.d/epinio.yaml#L89

Reply:
Version no longer needed and has been removed. It now now tracks whatever version the underlying epinio instance is, and value comes directly from call to `/info`